pkg/controller/incarnation/incarnation_controller.go

```diff
@@ -287,6 +284,16 @@ func (r *ResourceSyncer) Sync() error {
 	}
 	resourceStatus = append(resourceStatus, status)
 
+	var cpuTarget *int32
+	if !r.Instance.Spec.Scale.Resources.CPU.IsZero() {
+		cpuPercentage := int32(r.Instance.Spec.Scale.Resources.CPU.ScaledValue(resource.Milli) / 10)
```

Comment:
I'm not clear on what this does, but I want to make sure it gives a percent between 0-100. I'm almost positive that HPA compares targetCpuUtilizationPercentage against (currentCPU / scale.resource.CPU)